build.gradle

```diff
@@ -268,13 +268,13 @@ project(':cruise-control') {
     api 'org.apache.commons:commons-math3:3.6.1'
     api 'org.apache.httpcomponents:httpclient:4.5.13'
     api 'commons-codec:commons-codec:1.15'
-    api 'com.google.code.gson:gson:2.8.8'
+    api 'com.google.code.gson:gson:2.8.9'
     api "org.eclipse.jetty:jetty-server:${jettyVersion}"
-    api 'io.dropwizard.metrics:metrics-core:3.2.6'
+    api 'io.dropwizard.metrics:metrics-jmx:4.2.5'
```

Comment:
Migration from `metrics-core` `3.*` to `metrics-jmx` `4.*` introduces backwards incompatible changes (see https://github.com/linkedin/brooklin/pull/641 and the need for creating a workaround for it: https://github.com/linkedin/brooklin/pull/658).
Can we avoid this breaking change in this PR and add a comment to this line to indicate this to prevent unintentional migrations in the future?

If we would like to bump up this version, we need a similar workaround as linked above.

Reply:
you're right - I realized I am getting hasty at this.
I'll remove the upgrade of all dependencies (except log4j) in this PR to expedite the CVE fix and then I'll follow-up with a separate PR for the other deps upgrade.
Let me know if this works for you @efeg ?